plugins/wasm-cpp/extensions/jwt_auth/README_EN.md

```diff
@@ -88,12 +88,12 @@ The payload is where the actual information is stored. The details are defined b
 iss: The issuer of the token. This indicates who created the token and is a string value.
 sub: The subject identifier. This is the unique identifier for the end user provided by the issuer, and is no longer than 255 ASCII characters, and is case-sensitive within the issuer's scope.
 aud: The audience(s) of the token, which is an array of case-sensitive strings.
-exp: The expiration time of the token, which is a Unix timestamp in seconds, and any token that is issued after that time is invalid.
-iat: The time at which the token was issued, also a Unix timestamp in seconds.
-jti: The unique identifier for the token, and the value is unique for every token created by the issuer. It is usually a cryptographically random value to prevent conflicts. This component adds a random entropy that an attacker cannot obtain to the structured token, making it more difficult for the token to be guessed or replayed.
+exp: The expiration time of the token, after which the token will be invalidated, is a integer declaration representing the number of seconds since January 1, 1970.
```

Comment:
"an" integer.

Reply:
And I think Unix timestamp in seconds is good enough. Otherwise, you need to specifiy both time and timezone, which is 00:00 UTC.